src/windows/cppwinrt/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.UI.Xaml" version="2.3.191129002" targetFramework="native" />

```

Comment:
So the module itself doesn't depend on LottieWindows?

Reply:
Correct. It's up to the app to implement an ILottieSourceProvider and inject it into the module during setup. That allows the app to choose how to convert names or JSON into IAnimatedVisualSource, which could be through codegen, through LottieVisualSource, or some custom implementation.

The targets files greatly simplify this setup by auto-generating a provider and including it in the app's build (though you still need to new it and pass it into the module). For C++ apps, this provider only supports codegen and returns null for JSON requests. For C# apps, it supports both (using LottieVisualSource for JSON).

For codegen, LottieGen either needs to be installed globally on the system, or an MSBuild variable set to give its location. For LottieVisualSource, the app will need to add Lottie-Windows as a dependency.

The targets files have some initial documentation in them, but I can move it out into the README and add a section for Windows setup.